config/filter.d/centreon.conf

```diff
@@ -0,0 +1,9 @@
+# Fail2Ban filter for Centreon Web
+# Detecting unauthorized access to the Centreon Web portal
+# typically logged in /var/log/centreon/login.log
+
+[Init]
+datepattern = ^%%Y-%%m-%%d %%H:%%M:%%S
+
+[Definition]
+failregex = ^\[0-9-]*\|[0-9-]*\|\[0-9-]*|\[[^]]*\] \[<HOST>\] Authentication failed for '.+' :
```

Comment:
Firstly your anchored prefix `^\[0-9-]*...` does not work (because it searching open bracket `[`, because it's escaped `\[`)...
So the anchor does no matter and it's just the weird constellation (e. g. your forgotten escape by minus `-` char in sequence `[...]`, that means all chars starting with `9-`) what makes possible the RE matches by accident.
Also greedy catch-all `.+` at end is completely unneeded there (and one could use user name capturing RE).
So better to replace it with:
```diff
- failregex = ^\[0-9-]*\|[0-9-]*\|\[0-9-]*|\[[^]]*\] \[<HOST>\] Authentication failed for '.+' :
+ failregex = ^(?:\|-?\d+){3}\|\[[^\]]*\] \[<HOST>\] Authentication failed for '<F-USER>[^']+</F-USER>'
```
doing the same job, but it is anchored now.

Reply:
Thank you very much for your feedback 👍
I then just updated the rule according to your advices.